app/SearchApi/SearchApi.Web/Notifications/PersonSearchFinalizedCls.cs

```diff
@@ -0,0 +1,20 @@
+﻿using BcGov.Fams3.SearchApi.Contracts.PersonSearch;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SearchApi.Web.Notifications
+{
+    public class PersonSearchFinalizedCls : PersonSearchFinalized
```

Comment:
can we change the name to PersonSearchFinalizedEvent

Reply:
Sure